lua/fpp/server/ownability.lua

```diff
@@ -516,10 +520,7 @@ local function handleConstraintRemoved(ent)
     entMem[ent1] = true
     entMem[ent2] = true
```

Comment:
What if we use this `entMem`: instead of just storing the two entities, what about storing ALL the constrained entities in `entMem`?

That way, instead of calling the expensive once (or twice) per unique pair of entities, it would call it once per contraption.

I got this idea from a quick look. Would it work?

Reply:
I don't think this would work as entMem is checked before the timer is created.
Seems like the timer here is buggy though as it creates a unique timer that gets overwritten each time the function runs likely losing out on a lot of constraintRemovedTimer calls which should be made.

The current code has pretty much no lag impact even with calling it twice (tested with some workshop dupes with 300+ constraints and 300+ props) so performance should be fine as far as i can see.

Reply:
The profile says more than intuition. Thanks for taking a look :+1: